Source/WebKit/UIProcess/API/Swift/WebPage.swift

```diff
@@ -120,20 +125,125 @@ public class WebPage_v0 {
             createObservation(for: \.isLoading, backedBy: \.isLoading),
             createObservation(for: \.serverTrust, backedBy: \.serverTrust),
             createObservation(for: \.hasOnlySecureContent, backedBy: \.hasOnlySecureContent),
+            createObservation(for: \.isWritingToolsActive, backedBy: \.isWritingToolsActive),
             createObservation(for: \.themeColor, backedBy: \.themeColor),
         ]
     }
 
+    public static func handlesURLScheme(_ scheme: String) -> Bool {
+        WKWebView.handlesURLScheme(scheme)
+    }
+
     @discardableResult
     public func load(_ request: URLRequest) -> NavigationID? {
         backingWebView.load(request).map(NavigationID.init(_:))
     }
 
+    @discardableResult
+    public func load(_ data: Data, mimeType: String, characterEncoding: String.Encoding, baseURL: URL) -> NavigationID? {
```

Comment:
**For later:** I wonder if it’s better to use `mimeType: String` or `contentType: UTType` here. We didn’t have a strongly typed framework for this back when the `WKWebView` API was added, but I’m also unsure if that’d be a usability regression for providing parameters like `text/plain;charset=UTF-8`.

Reply:
I do agree that we should eventually consider a `UTType` argument here.